index.html

```diff
@@ -22,7 +22,7 @@
       <span class="section-1-star">*</span>
     </div>
   </section>
-  <img class="section-1-lights" src="assets/images/img1@3x.png" />
+  <img class="section-1-lights" alt="Electronic transactions for all" src="assets/images/img1@3x.png" />
```

Comment:
Unsure about this alt text maybe something more general. Isn't the primary purpose of `alt` text to describe the image to visitors who are unable to see them?

Reply:
You're right. Thanks!

Reply:
something like `Protect yourself against electronic surveillance` ? Unsure trying to describe that imagine is hard.

Reply:
Once this is fixed we can merge this

Reply:
Done. Thanks!